.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "kiota-abstractions-dotnet"]
```

Comment:
why are we relying on a module here and not a package reference?

Reply:
We need the powershell scripts for CI

Reply:
adding a submodule just for a powerhsell script feels a bit overkill, let's duplicate it?